src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -94,16 +94,16 @@ public void equals() {
         // null -> returns false
         assertFalse(deleteFirstCommand.equals(null));
 
-        // different person -> returns false
+        // different student -> returns false
         assertFalse(deleteFirstCommand.equals(deleteSecondCommand));
     }
 
     /**
      * Updates {@code model}'s filtered list to show no one.
      */
     private void showNoPerson(Model model) {
```

Comment:
```suggestion
    private void showNoStudent(Model model) {
```
A few of the other tests have some parts that are not refactored yet, but I think can just leave it ba.